web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties

```diff
@@ -1037,11 +1024,7 @@ lockout_changes_have_been_saved=The changes you made to the Lockout Configuratio
 job_ran_for=Job ran for
 job_ran_but_no_files=Job ran but no real files found.
 job_failure_for=Job FAILURE for
-your_job_ran_success_html=,<br/>Your job ran successfully with the following messages returned.
 please_review_the_data_html=Review the data, or modify the files accordingly so that they can be imported.<br/>
-unreadable_file=Unreadable File
```

Comment:
This line is referenced in ImportSpringJob.java.  However the reference is in dead code in the class.  For consistency, either the string should be left here or the dead code in ImportSpringJob.java should also be removed.